### PR TITLE
Fixes #55 - moves chaos examplpe into a separate build

### LINKED ISSUE #55: Extract example into a separate build
Having the example in the same structure as the chaos build is not convenient.

### PR DESCRIPTION
**Pull request by alexanderweber**
Like suggested in #55, examples are moved into a separate build.

### CODE REVIEW

**Comment by alexanderweber** [comment]
@aquamatthias I moved the example to a separate build as discussed before. Will do additional examples (including actors) in a separate PR.

File: project/build.scala [hunk 1]
```diff
@@ -11,6 +11,14 @@ import sbtrelease.ReleasePlugin._
 import scalariform.formatter.preferences._
 
 object ChaosBuild extends Build {
+  lazy val chaosExamples: Project = Project(
+    id = "chaos-examples",
+    base = file("chaos-examples"),
+    settings = baseSettings ++
+      formatSettings ++
```
Comment (aquamatthias) [inline project/build.scala]: please add revolverSettings

**Comment by aquamatthias** [comment]
1) The README and help.md files point to `/foo`, but you changed it to `/persons`. Please also adjust documentation.
2) Perhaps rename `chaos-examples` to just `example`. We are already in chaos and it is one example not multiple.

Otherwise looks great! Thanks!

**Comment by alexanderweber** [comment]
I had some troubles merging that one to the previous branch, but now it should be all fine I guess.

**Comment by aquamatthias** [comment]
1) One open feedback (revolverSettings)
2) Please rebase on master
3) I will merge :)

File: project/build.scala [hunk 2]
```diff
@@ -11,6 +11,15 @@ import sbtrelease.ReleasePlugin._
...
+      publishSettings ++
+      revolverSettings
```
Comment (alexanderweber) [inline project/build.scala]: @aquamatthias I did add revolverSettings right here. Did you mean some other place and I am misunderstanding it?

**Comment by alexanderweber** [comment]
2) I did a rebase on master using IntelliJ, but I am anything but sure, that it did the right thing. Would be glad if you could assist me there f2f today.

**Pull request closed without merging**